File: samples/ntlm-module-sample/app/Global.scala
import play.api.mvc.WithFilters
import dk.guldner.play.ntlm.NtlmFilter
import play.api.GlobalSettings
import play.api.Application
import dk.guldner.play.ntlm.Conf

object Global extends WithFilters(NtlmFilter()) with GlobalSettings {

  override def onStart(app: Application) {
    play.api.Logger.info("Application has started")
    //Conf.logNtlmModuleConfiguration
  }  
} 

File: samples/ntlm-module-samplej/app/Global.java
import play.GlobalSettings;
import play.api.mvc.EssentialFilter;
import dk.guldner.play.ntlm.*;


public class Global extends GlobalSettings {

	@SuppressWarnings({ "rawtypes", "unchecked" })
	@Override
	public <T extends EssentialFilter> Class<T>[] filters() {
		Class[] filters = {NtlmFilter.class};
		return filters;
	}
}
